modules/claims_api/spec/fixtures/usability_testing_claims.json

```diff
@@ -4,35 +4,35 @@
             "id": "600098193",
             "type": "evss_claims",
             "attributes": {
-                "evssId": 600098193,
-                "dateFiled": "2017-05-02",
-                "minEstDate": null,
-                "maxEstDate": null,
+                "evss_id": 600098193,
```

Comment:
why was this changed to snake_case?

Reply:
Because [the schema file](https://github.com/department-of-veterans-affairs/vets-api/blob/master/spec/support/schemas/claims_api/claims.json#L25) that `rswag` is using to compare the response to, has the attributes defined using snake_case.  `rswag` fails the testing because stuff like `evssId` != `evss_id`.

Reply:
gotcha, ok, yeah, we'll end up needing new schema files for v2 for that sort of stuff when we actually go and implement it.